test/.nbval_sanitize.cfg

```diff
@@ -10,3 +10,29 @@ replace: $
 
 regex: \s*\$\s*
 replace: $
+
+[sympy printing changes]
+regex: \\left \(
+replace: \\left(
+regex: \\right \)
+replace: \\right)
+regex: \\left \[
+replace: \\left[
+regex: \\right \]
+replace: \\right]
+
+# note hacks to avoid trailing spaces which editors may trim
+regex: , \\quad( )
+replace: , \ \1
+
+# sympy 1.5 seems to reorder a few additions compared to 1.3
+regex: \\begin\{matrix}- \\frac\{\\sqrt\{2}}\{4} \+ \\frac\{1}\{2}\\end\{matrix}
+replace: \\begin{matrix}\\frac{1}{2} - \\frac{\\sqrt{2}}{4}\\end{matrix}
+regex: =( +)- \\frac\{\\sqrt\{2}}\{4} \+ \\frac\{1}\{2}(?=\$| \\end\{equation\*})
+replace: =\1\\frac{1}{2} - \\frac{\\sqrt{2}}{4}
+
+regex: \{- \\left\( e_\{E}\\cdot e_\{B}\\right\) \^\{2} \+ 1\}
+replace: {1 - \\left( e_{E}\\cdot e_{B}\\right) ^{2}}
+
+regex: eu\.ev_r = -1/2 \+ 1/2(?=\n)
+replace: eu.ev_r = 1/2 - 1/2
```

Comment:
Aha, you use this to make both version happy

Reply:
We better use the 1.5 output as the default, and replace the old one to the new one here?

Reply:
We can do that in a follow-up

The sanitizations apply both to the expected and actual result - so will keep working when we regenerate the notebook with 1.5.0